### ISSUE TITLE
Bug on _export_rows

### ISSUE THREAD

**Issue by stanevicius1** [gmarczynski/odoo-web-progress#5]
hi,
change this place:
https://github.com/gmarczynski/odoo-web-progress/blob/c8627aadbc2a27c35c233aeb0f02d0b63fd6fb16/web_progress/models/base.py#L164

to:
`ret += super(Base, sub)._export_rows(fields)`

or create monkey patching like that:
```
@api.model
def load_views(self, views, options=None, my_new_param=True):
    .......

models.BaseModel.load_views = load_views

```

**Comment by gmarczynski** [comment]
Hi, is this for 11.0 or 12.0?
What is the error scenario?

**Comment by stanevicius1** [comment]
hi,
11.0 but i guess in both of version should be the same if the same logic used. I'm suggesting to use monkey patching then everywhere will be needed parameter.

**Comment by gmarczynski** [comment]
Thx. What is the error scenario? Why do you think this is a bug?

**Comment by stanevicius1** [comment]
```
/web_progress/models/base.py", line 164, in _export_rows
    ret += super(Base, sub)._export_rows(fields, batch_invalidate=False)
TypeError: _export_rows() got an unexpected keyword argument 'batch_invalidate'
```
from brakepoint:
```
pycharm-community-2019.2/helpers/pydev/_pydevd_bundle/pydevd_exec2.py", line 3, in Exec
    exec(exp, global_vars, local_vars)
  File "<input>", line 1, in <module>
TypeError: _export_rows() got an unexpected keyword argument 'batch_invalidate'
```

```
>>>sub
sale.order(3741308, 3741306, 3741307)
```

```
fields
[['partner_id', 'id']]
```

**_export_rows** is on:
```
@pycompat.implements_to_string
class BaseModel(MetaModel('DummyModel', (object,), {'_register': False})):
```
that's why i'm suggesting monkey patching instead inheritance of base model.

**Comment by gmarczynski** [comment]
I ma a bit confused, because the signature of `BaseModel._export_rows` for Odoo 11.0 is the following:

https://github.com/odoo/odoo/blob/845eecdfa7b27d84862e94f4b63d40a851e0c410/odoo/models.py#L692
```(python)
@api.multi
def _export_rows(self, fields, batch_invalidate=True):
```

There is clearly a keyword parameter `batch_invalidate`, so how can you get a type error?

Maybe you have some other module installed that overrides `_export_rows` without this parameter? 

Can you modify the code locally as you propose, set a breakpoint, step into the call and show me what method is actually called?

Thx.

**Comment by stanevicius1** [comment]
It looks like my odoo was installed before this fix on odoo server:
[FIX] export performance regression from d28f8f7
https://github.com/odoo/odoo/commit/1e6da402d714ff6227125c3b41a4b8805464a953
